Program.cs

```diff
@@ -1,12 +1,71 @@
-﻿using System;
+﻿using PhoneBook.StaticHelpers;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
 
 namespace PhoneBook
 {
-    class Program
+    static class Program
     {
+        private static string formatStrig = "NNNNNNNNN";
+        public static string path;
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.ReadLine();
+            Runer();
+        }
+
+        private static void Runer()
+        {
+            bool flag = true;
+            while (flag)
+            {
+                Console.WriteLine("PhoneBook Program:");
+                Console.WriteLine();
+                path = @"..\\..\\..\\Samples\\FirstPhoneBook.txt";
+                string info = FileHelpers.Reader(path);
+                Console.WriteLine(info);
+                string[] sub = info.Split(' ', '\r', '\n');
+                int j = 0;
+                int count = 0;
+                List<PersonInfo> persons = new List<PersonInfo>();
+                for (int i = 0; i < sub.Length; i++)
+                {
+                    if (sub[i] == "" || i == sub.Length - 1)
+                    {
+                        if (i - j == 4 || (i == sub.Length - 1 && i - j == 3))
+                        {
+                            persons.Add(new PersonInfo());
+                            persons[count].Name = sub[j];
+                            persons[count].Surname = sub[j + 1];
+                            persons[count].Number = sub[j + 3];
+                            if (!(sub[j + 2] == ":" || sub[j + 2] == "-"))
```

Comment:
Hardcoded separators. What if in future we will have instead of ":" or "-" have a 20 valid separators